src/flow.ts

```diff
@@ -328,8 +328,8 @@ export class Flow {
     return Tables.clefProperties(clef);
   }
   // eslint-disable-next-line
-  static keyProperties(key: string, clef?: string, params?: any): any {
-    return Tables.keyProperties(key, clef, params);
+  static keyProperties(duration:string, key: string, clef?: string, params?: any): any {
```

Comment:
This changes the method signature. If you need to add this parameter, we should do it in a backwards compatible way if at all possible (unless there is a very compelling reason to change the method signature).

Reply:
In VexFlow there are two methods to specify the note heads:
- using `type` which affects all the note heads in a chord (this method takes into account the duration to select the appropriate note head ie: black, half...)
- using and additional parameter on the 'key' string (this was only thought for custom noteheads and does not take into accout the duration)
however it is possible to have different types of note heads that should take into account the duration.

As a consequence I made the second method duration aware.

Reply:
Done. Changed to addtional param.